src/index.jsx

```diff
@@ -34,12 +34,36 @@ const apiClient = getAuthenticatedAPIClient({
   loggingService: NewRelicLoggingService,
 });
 
+let callbackAccessToken = null;
+
+/**
+ * Temporary check of auth state vs callback access token to help debug and fix any
+ * potential problems.
+ */
+function checkAuthState(functionName, authenticationState) {
+  const userIdsMatch = !!(
+    callbackAccessToken && callbackAccessToken.user_id &&
+    authenticationState && authenticationState.authentication &&
+    authenticationState.authentication.userId &&
+    callbackAccessToken.user_id === authenticationState.authentication.userId
+  );
+  const customAttributes = {
+    functionName,
+    callbackAccessToken,
+    authenticationState,
```

Comment:
Do you need to log the entire `callbackAccessToken` and `authenticationState` object values?  Any PII concerns there?

Reply:
Was just trying to get something done quick and didn’t know what would be helpful for debugging. We’ve done this before. I don’t think email and username to NewRelic temporarily will be an issue. If you are concerned I can edit down further.